scripts/test-files/windows2k22-installer-pipelinerun.yaml

```diff
@@ -1,13 +1,19 @@
 apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
-  generateName: windows2k22-installer-run-
+  generateName: windows2k22-installer-
   labels:
-    pipelinerun: windows2k22--installer-run
+    pipelinerun: windows2k22-installer-run
 spec:
   params:
   - name: winImageDownloadURL
     value: http://http-server/disk.img
+  - name: autounattendConfigMapName
+    value: windows2022-autounattend
```

Comment:
Might rname that config map too to `2k22`?

Reply:
fixed